incubator/jaeger/README.md

```diff
@@ -129,54 +133,58 @@ The command removes all the Kubernetes components associated with the chart and
+| `collector.tchannelPort`           | Jaeger Agent port for thrift        |  14267                                 |
+| `collector.type`                   | Service type                        |  ClusterIP                             |
+| `collector.zipkinPort`             | Zipkin port for JSON/thrift HTTP    |  9411                                  |
+| `hotrod.enabled`                   | Enables the Hotrod demo app         |  false                                 |
+| `spark.enabled`                    | Enables the dependencies job        |  false                                 |
+| `spark.image`                      | Image for the dependencies job      |  jaegertracing/spark-dependencies      |
+| `spark.tag`                        | Tag of the dependencies job image   |  latest                                |
+| `spark.pullPolicy`                 | Image pull policy of the deps image |  Always                                |
+| `spark.schedule`                   | Schedule of the cron job            |  "*/12 * * * *"                        |
+| `query.annotationsPod`             | Annotations for Query UI            |  nil                                   |
+| `query.annotationsSvc`             | Annotations for Query SVC           |  nil                                   |
+| `query.cmdlineParams`              | Additional command line parameters  |  nil                                   |
+| `query.image`                      | Image for Jaeger Query UI           |  jaegertracing/jaeger-query            |
+| `query.ingress.enabled`            | Allow external traffic access       |  false                                 |
+| `query.pullPolicy`                 | Query UI image pullPolicy           |  IfNotPresent                          |
+| `query.queryPort`                  | External accessible port            |  80                                    |
+| `query.tag`                        | Image tag/version                   |  0.6                                   |
+| `query.targetPort`                 | Internal Query UI port              |  16686                                 |
+| `query.type`                       | Service type                        |  ClusterIP                             |
+| `schema.annotations`               | Annotations for the schema job      |  nil                                   |
+| `schema.image`                     | Image to setup cassandra schema     |  jaegertracing/jaeger-cassandra-schema |
+| `schema.mode`                      | Schema mode (prod or test)          |  prod                                  |
+| `schema.pullPolicy`                | Schema image pullPolicy             |  IfNotPresent                          |
+| `schema.tag`                       | Image tag/version                   |  0.6                                   |
+|------------------------------------|-------------------------------------|----------------------------------------|
```

Comment:
The new `storage.**` parameters are not listed here

Reply:
I'll fix it shortly

Reply:
done